<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_deposit_calc</class>
 <widget class="QWidget" name="form_deposit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
	background-color: gray;
	border: 1px solid gray;
	color: white;;
}
QWidget  {
	background-color: blue;
}
QDoubleSpinBox {
	background-color: gray;
	border: 1px solid gray;
	color: white;;
}
QSpinBox {
	background-color: gray;
	border: 1px solid gray;
	color: white;;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>532</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="2" colspan="2">
     <widget class="QRadioButton" name="CapitalizationEndOfTerm">
      <property name="text">
       <string>end of term</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QDoubleSpinBox" name="Placements_2">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="labelWithdrawals">
      <property name="text">
       <string>Withdrawals:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Deposit percent:</string>
      </property>
     </widget>
    </item>
    <item row="16" column="3">
     <widget class="QLineEdit" name="TaxAmount">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: gray;
	border: 1px solid gray;
	color: #ffffff;;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QComboBox" name="ComboboxPlacements">
      <property name="styleSheet">
       <string>QComboBox{
	Background: green;
	border: 1px green;
}

</string>
      </property>
      <item>
       <property name="text">
        <string>monthly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>quarterly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>once in 6 months</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yearly</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="17" column="3">
     <widget class="QLineEdit" name="DepositClosingAmount">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: gray;
	border: 1px solid gray;
	color: #ffffff;;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QRadioButton" name="CapitalizationQuarter">
      <property name="text">
       <string>quarterly</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="2">
     <widget class="QLabel" name="DepoCloseAmount">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Deposit closing amount: </string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Deposit amount:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="LabelPlacements">
      <property name="text">
       <string>Placements:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Payment frequency:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QComboBox" name="Payment_frequency">
      <property name="styleSheet">
       <string notr="true">QComboBox{
	Background: green;
	border: 1px green;
}

</string>
      </property>
      <item>
       <property name="text">
        <string>Monthly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>End of term</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="QLabel" name="TotalInterests">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Accumulated interests:</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="2">
     <widget class="QLabel" name="Tax_Amount">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tax amount:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1" colspan="3">
     <widget class="QPushButton" name="CALCULATE">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #006400;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #008000;
	border: 1px solid gray;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>CALCULATE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Tax rate:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QRadioButton" name="CapitalizationMonth">
      <property name="text">
       <string>monthly</string>
      </property>
     </widget>
    </item>
    <item row="15" column="3">
     <widget class="QLineEdit" name="AccruedInterests">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: gray;
	border: 1px solid gray;
	color: #ffffff;;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Deposit term:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QComboBox" name="comboBoxWithdrawals">
      <property name="styleSheet">
       <string notr="true">QComboBox{
	Background: green;
	border: 1px green;
}

</string>
      </property>
      <item>
       <property name="text">
        <string>monthly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>quarterly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>once  in 6 months</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yearly</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="2" colspan="2">
     <widget class="QRadioButton" name="CapitalizationHalfYear">
      <property name="text">
       <string>half a year</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="Capitalization">
      <property name="text">
       <string>Capitalization: </string>
      </property>
     </widget>
    </item>
    <item row="12" column="3">
     <widget class="QDoubleSpinBox" name="Withdrawals_2">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QDoubleSpinBox" name="Tax_rate_2">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QDoubleSpinBox" name="Deposit_percent_2">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="Deposit_term_2">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <number>600</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QDoubleSpinBox" name="Deposit_amount_2">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="maximum">
       <double>90000000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
